### DISCUSSION TITLE
How can I deploy master to a live website?

### DISCUSSION THREAD

**Discussion by NotLazy** [gogs/gogs#6849] [Q&A] [answered]
Looking to use gogs for auditting and version control on a handful of websites, but I can't figure out where to put the root directory for the website.

How can I deploy to a directory on my server via gogs?

**Comment by unknwon** [comment, marked as answer]
You can't directly archive this via Gogs today, the recommended way is to set up webhooks and send to something you own and executing deploy scripts/commands. It is also possible to use Git hook if you're site admin (in **Repository > Settings > Git Hooks > post-receive**):

<img width="1132" alt="CleanShot 2022-03-16 at 12 47 55@2x" src="https://user-images.githubusercontent.com/2946214/158518632-9f5ebc1f-16f9-488c-8227-473bd64563d0.png">

But it is known to be hard to debug with.
